src/MyClassSeventh2.java

```diff
@@ -0,0 +1,25 @@
+public class MyClassSeventh2 {
+    public static boolean isValidUkrainianPhoneNumber(String number) {
+        if (!number.startsWith("+380")) { //But we can call 0959493455, without +38. Idk why
```

Comment:
why not implement the task using regex:
 
 public static void validateUkrainianNumbers(List<String> mobileNumbers) {
        String regex = "^\\+380\\d{9}$";
        Pattern pattern = Pattern.compile(regex);

        for (String mobnumber : mobileNumbers) {
            Matcher matcher = pattern.matcher(mobnumber);
            System.out.println(mobnumber + " : " + matcher.matches());
        }